Resources/Prototypes/Entities/Objects/Specific/Janitorial/trashbag.yml

```diff
@@ -26,6 +27,7 @@
   - type: Item
     sprite: Objects/Specific/Janitorial/trashbag.rsi
     HeldPrefix: blue
+    size: 125
```

Comment:
Very minor but blue trash bag could parent normal trash bag so you don't need to repeat the storage component. Saves like 4 lines.

Reply:
That one passed me right by, thanks. changed it to use trash bag as parent.

![image](https://user-images.githubusercontent.com/47410468/147413077-b52282c1-11ee-489e-9194-a18c20631582.png)